docs/data/load-balanced-gridftp.md

```diff
@@ -334,23 +327,16 @@ Using Your Load Balanced GridFTP System
 
 ### Using GridFTP
 
-On the GridFTP servers, arptables is the only additional service required for running a load-balanced GridFTP system. The name of the arptables service depends on the version of your host OS:
-
-1. Select the appropriate RPM:
-
-    | **If your operating system version is...** | **Then use the following package(s)...** | 
-    | ---------------------------------------- | -------------------------------------- |
-    | Enterprise Linux 6 | `arptables_jf` | 
-    | Enterprise Linux 7 | `arptables` | 
+On the GridFTP servers, arptables is the only additional service required for running a load-balanced GridFTP system.
 
 1. Manage the service with the following commands:
 
     | **To ...** | **On EL 6, run the command...** | **On EL 7, run the command...** | 
     | -------- | ----------------------------- | ----------------------------- |
-    | Start a service | `service SERVICE-NAME start` | `systemctl start SERVICE-NAME` | 
-    | Stop a service | `service SERVICE-NAME stop` | `systemctl stop SERVICE-NAME` |
-    | Enable a service to start during boot | `chkconfig SERVICE-NAME on` | `systemctl enable SERVICE-NAME` | 
-    | Disable a service from starting during boot | `chkconfig SERVICE-NAME off` | `systemctl disable SERVICE-NAME` |
+    | Start the service | `service arptables start` | `systemctl start arptables` | 
+    | Stop atheservice | `service arptables stop` | `systemctl stop arptables` |
```

Comment:
I think you missed this table?
Also replace `Stop atheservice` with `Stop the service`

Reply:
Good catch. Fixed